wsi/physicaldisplay.cpp

```diff
@@ -405,6 +405,19 @@ void PhysicalDisplay::SetExplicitSyncSupport(bool disable_explicit_sync) {
   display_queue_->SetExplicitSyncSupport(disable_explicit_sync);
 }
 
+void PhysicalDisplay::SetVideoColor(HWCColorControl color, float value) {
+  display_queue_->SetVideoColor(color, value);
```

Comment:
You will need same plumbing for Logical, Mosaic and Nested Display. LogicalDisplay might be bit tricky as same PhysicalDisplay is going to be split into multiple displays but at OS level Logical is still an individual Physical Display, so you might be making duplicate calls to same physical display eventually in hwcservice. This is k since it shouldn't add any extra work when compositing the layers though.